### ISSUE TITLE
Feature request: Use system font on android

### ISSUE THREAD

**Issue by xronin01** [localsend/localsend#467]
### Is your feature request related to a problem? Please describe.

Hi,

Would it be possible to use the default system font ?

Thanks !

### Describe the solution you'd like

Hi,

Would it be possible to use the default system font ?

Thanks !

**Comment by Sobaaaan** [comment]
I don't think that Flutter allows the applications to do that, you can follow [flutter #48381](https://github.com/flutter/flutter/issues/48381) for further details. 
Although it might be possible to add different fonts option in settings where a user can select a font of their choice.

**Comment by DwainZwerg** [comment]
@Tienisto There is a plugin: https://github.com/re7gog/android_system_font
